onecore_maintenance_extension/static/src/views/maintenance_request_item.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates xml:space="preserve">
+    <t t-name="onecore_maintenance_extension.maintenance_request_item">
+        <span class="oe_grey" t-if="record.address.raw_value">
+            <span t-if="record.address.raw_value">
+                <t t-esc="record.address.raw_value"/>
+            </span>
+            <br/>
+        </span>
+        <span t-if="record.space_caption.raw_value || record.rental_property_id.raw_value">
+            <t t-esc="record.space_caption.raw_value"/>
+            /
+            <t t-esc="record.rental_property_id.raw_value"/>
+            <br/>
+        </span>
+        <span t-if="record.space_caption.raw_value != 'Tvättstuga'">
```

Comment:
@johanneskarlsson Had to change from `invisible` to `t-if` here.

Reply:
That works too 🙌